features/javascript/response.feature

```diff
@@ -0,0 +1,76 @@
+@javascript
+Feature: Response validation
+
+  Background:
+    Given you call:
+    """
+    const gavel = require('gavel');
+    """
+    And you define following HTTP response object:
+    """
+    const real = {
+      "statusCode": "200",
+      "statusMessage": "OK",
```

Comment:
The actual response includes a `statusMessage` of `OK` but the expected response does not. I don't see any validation in the result, should there be any output that states that statusMessage differs? Although status message is only in HTTP 1 and removed in HTTP 2 so server implementation could not include this.

Reply:
I believe this is because the validation performed by Gavel is not strict, meaning it allows for actual HTTP message to include _more_ fields than in the expected message. This is why `statusMessage` in the actual and its lack in the expected is a valid behavior.

Reply:
It seems that the validation of `statusMessage` has been lost during the `gavel.js` refactoring. Should we support this deprecated property on HTTP messages?